path/config.go

```diff
@@ -0,0 +1,28 @@
+package path
+
+import "path/filepath"
+
+// Config returns the path of config.json
+func Config() string {
+	return configPath("config.json")
+}
+
+// LocalKeys returns path of the local private keys or certificate in the localkeys directory
+func Localkeys(keyName string) string {
+	return configPath(filepath.Join("localkeys", keyName))
+}
+
+// SigningKeys return the path of signingkeys.json files
+func SigningKeys() string {
+	return configPath("signingkeys.json")
+}
+
+// TrustPolicy returns the path of trustpolicy.json file
+func TrustPolicy() string {
+	return configPath("trustpolicy.json")
+}
+
+// TrustStore returns the path of x509 trust store certificate
+func X509TrustStoreCerts(identity, namedStore, cert string) string {
```

Comment:
This doesn't need to resolve paths for certificates, only named stores is sufficient. See [this](https://github.com/notaryproject/notation-go/blob/main/verification/store.go#L22) for how we will read x509 trust stores.

Also, i would rename the identity variable to `prefix` to be consistent with https://github.com/notaryproject/notation-go/pull/72